### DISCUSSION TITLE
DateTimePicker with option Multiple dates selected

### DISCUSSION THREAD

**Discussion by anna777x** [dotnetcore/BootstrapBlazor#3195] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

I'm trying to find DateTimePicker with option Multiple. I like a lot of components of  BootstrapBlazor library. So,  is it possible to extend BootstrapBlazor Datetime Picker to have this option?

or maybe you know quick solution how can I extend existing DateTimePicker component for getting such result?

### Describe the solution you'd like

It can look like in this library 
https://blazor.syncfusion.com/documentation/calendar/multi-select
or 
https://blazorise.com/docs/components/date-picker    ->   Multiple example

### Additional context

please let me know any ideas.

**Comment by bb-auto** [comment]
@anna777x Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment]
I think it isn't easy to extend the existing components because your requirement seems to require multiple values, but the current components only have one value. So let us create a new component. What do you think?

**Reply by anna777x** [reply]
I agree. I need this functionality. It shouldn't be existing component.

**Reply by anna777x** [reply]
When can I  have it as part of BootstrapBlazor on your point?

**Reply by ArgoZhang** [reply]
I think the next milestone is enough. If you can help implement this component, I think it will be faster
